pandas/tests/arrays/boolean/test_comparison.py

```diff
@@ -0,0 +1,127 @@
+import numpy as np
+import pytest
+
+import pandas as pd
+import pandas._testing as tm
+from pandas.arrays import BooleanArray
+from pandas.tests.extension.base import BaseOpsUtil
+
+
+def make_data():
+    return [True, False] * 4 + [np.nan] + [True, False] * 44 + [np.nan] + [True, False]
+
+
+@pytest.fixture
+def dtype():
+    return pd.BooleanDtype()
+
+
+@pytest.fixture
+def data(dtype):
+    return pd.array(make_data(), dtype=dtype)
+
+
+class TestComparisonOps(BaseOpsUtil):
```

Comment:
should these be parametrized over EA/Series/DataFrame? (and eventually ExtensionIndex)?

Reply:
Perhaps, I'm trying to avoid making any actual logic changes to tests here though since the PR is already quite big